roles/openshift_logging/defaults/main.yml

```diff
@@ -160,8 +169,9 @@ openshift_logging_mux_allow_external: False
 openshift_logging_use_mux: "{{ openshift_logging_mux_allow_external | default(False) }}"
 openshift_logging_mux_hostname: "{{ 'mux.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true)) }}"
 openshift_logging_mux_port: 24284
-openshift_logging_mux_cpu_limit: 500m
+openshift_logging_mux_cpu_limit: null
 openshift_logging_mux_memory_limit: 1Gi
```

Comment:
This doesn't match  the value of 512Mi in roles/openshift_logging_mux/defaults/main.yml.  Should it?

Reply:
@mffiedler, it should.  I'll fix it.